helm/dagster/schema/schema_tests/test_dagit.py

```diff
@@ -49,6 +49,9 @@ def workspace_configmap_helm_template() -> HelmTemplate:
     )
 
 
+# Parametrizing "webserver_field" here tests backcompat. `dagit` was the old field name.
+# This parametrization can be removed in 2.0.
+@pytest.mark.parametrize("webserver_field", ["dagsterWebserver", "dagit"])
```

Comment:
just this one? maybe slap this parametrize on a few more like [2]

Reply:
done